projects/ids-enterprise-ng/src/lib/accordion/soho-accordion-header.component.ts

```diff
@@ -24,8 +24,12 @@ export class SohoAccordionHeaderComponent implements AfterViewInit {
   jQueryElement: JQuery;
 
   @HostBinding('class.is-disabled') isDisabled = false;
-  @HostBinding('style.display') get isBlockDisplay() { return 'block'; }
-  @HostBinding('class.accordion-header') get isAccordionHeader() { return true; }
+  @HostBinding('style.display') get isBlockDisplay() {
+ return 'block';
+}
+  @HostBinding('class.accordion-header') get isAccordionHeader() {
```

Comment:
Should this be:
```
  @HostBinding('class.accordion-header') get isAccordionHeader() {
    return true;
  }
```

If thats a rule im not sure why it doesnt line up.

Reply:
Yes - it only complains about the lack of newlines, I thought I fixed the formatting when I added the newline in but after doing 200 of them I must have missed one ;-)

Reply:
I just mean it looks like its like this....
```
  @HostBinding('style.display') get isBlockDisplay() {
 return 'block';
}
```

but maybe should be 
```
  @HostBinding('style.display') get isBlockDisplay() {
    return 'block';
   }
```

But no need to go back and find all of these... Maybe just fix a couple if you see them

Reply:
Fixed